### DISCUSSION TITLE
Sample configuration file for chaostoolkit-kubernetes

### DISCUSSION THREAD

**Discussion by nitinbhat** [chaostoolkit/chaostoolkit#308] [Q&A]
Can any one share sample configuration which has all steady-state-hypothesis, runtime, hypothesis, rollback, controls, activities, action etc. including settings.yaml and how to use the same ?
Documentation has minimal configuration and I am facing hard time to build a configuration file as there is no reference on how to make use of certain features

**Comment by cdsre** [comment]
Hi @nitinbhat, your question as it stands is very vague. Do you have a particular scenario in mind. The docs for [CTK-kubernetes](https://chaostoolkit.org/drivers/kubernetes/) focus on the functionality of K8 actions, probes etc. There is an expectation that the user is already familiar with the [CTK experiment](https://chaostoolkit.org/reference/api/experiment/) template. 

The K8 extension is pretty vast so the best way to get any support would be to define a specific example to work to achieve. Once you have that and a working scenario you will then find it easier to on board other features and functionality to enrich your tests.

Also you will likely get more of an active response from the slack group as not a lot of the community is actively watching the discussions pages. https://chaostoolkit.slack.com/archives/C77UZUY82

**Reply by nitinbhat** [reply]
Hi @cdsre 
I am new to chaostoolkit and I am specifically interested in chaotoolkit-kubernetes. I have referred [CTK-kubernetes](https://chaostoolkit.org/drivers/kubernetes/) and [CTK experiment](https://chaostoolkit.org/reference/api/experiment/) and have built some configuration but I am also interested in understanding how to configure rollbacks or controls etc. or how to run multiple experiments together, how to configure settings for which I couldn't find examples in the documentation page.

Thanks for sharing link to Slack group. I shall check for examples there.

**Reply by cdsre** [reply]
So the best thing to do is share your initial configuration and then focus on adding the next feature like say roll back, or control etc. Then once thats working focus on the next part. That way your operating an MVP approach and using continuous improvement ensuring you get value at every step rather than biting off a large bit of work that takes months to produce.

Personally I am not familiar with the CTK-Kubernetes extension as I have not used it directly. but again asking on slack will probably get you a faster response. But just try to be focused and add 1 bit at a time.
